shared/fs/header/breadcrumb-popup.js

```diff
@@ -28,14 +29,13 @@ const BreadcrumbPopupMenu = ({routeProps}: PopupMenuProps) => {
   const isTeamPath = routeProps.get('isTeamPath')
   const items = routeProps.get('items')
   const onOpenBreadcrumb = routeProps.get('onOpenBreadcrumb')
+  const onHidden = routeProps.get('onHidden')
   const popupItems = items.map((i, idx) => ({
-    onClick: () => {
-      onOpenBreadcrumb(i.path)
-    },
+    onClick: onOpenBreadcrumb(i.path),
```

Comment:
we rarely do something like this. why make a function return a function here

Reply:
OK so we do this in order to support both the breadcrumbs in the dropdown and in the top-level menu with the same function.

Reply:
Actually nvmind I think I can do this your way too.